### DISCUSSION TITLE
Cube-lrwan execution error

### DISCUSSION THREAD

**Discussion by tcpipchip** [Lora-net/LoRaMac-node#1236] [Q&A] [answered]
Sir,
I testing the STM32L0 + SX1276 with en.i-cube_lrwan_v2.1.0 lib
I compiled the AT COMMANDS example on KEIL and then with STM32 IDE too, but during execution i am always getting the following error in both
APP_VERSION:    V1.1.0
MW_LORAWAN_VERSION: V2.3.0
MW_RADIO_VERSION:  V1.1.0
0s000:error: The Region is not defined in the MW: set lorawan_conf.h accordingly
it´s weird, because in lorawan_conf.h i defined
 
/* Region ------------------------------------*/
/* the region listed here will be linked in the MW code */
/* the application (on sys_conf.h) shall just configure one region at the time */
/*#define REGION_AS923*/
#define REGION_AU915
/*#define REGION_CN470*/
/*#define REGION_CN779*/
/*#define REGION_EU433*/
/*#define REGION_EU868*/
/*#define REGION_KR920*/
/*#define REGION_IN865*/
/*#define REGION_US915*/
/*#define REGION_RU864*/
 
Some tip ?
 
image 
Thank you!!!!!

**Comment by mluis1** [comment, marked as answer]
This project does not provide an example using AT commands.
This question is related to ST fork of this project. Please, ask ST for support.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Comment by tcpipchip** [comment]
i am sorry! thanks
